eZ/Publish/Core/MVC/Symfony/SiteAccess/SiteAccessProviderInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\SiteAccess;
+
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use Traversable;
+
+/**
+ * @internal
+ */
+interface SiteAccessProviderInterface
```

Comment:
AFAIR this is supposed to be used by EE SiteFactory. How does the code for that look like if it's internal? Is the extension somewhere else?